api/package.json

```diff
@@ -26,7 +26,7 @@
     "fast-csv": "^2.4.1",
     "google-libphonenumber": "^3.1.3",
     "helmet": "^3.12.1",
-    "http-proxy": "^1.16.2",
+    "http-proxy": "^1.17.0",
```

Comment:
This version allows intercepting the proxy response, which is necessary for sending Couch-like responses to restricted `_bulk_docs`.